src/test/chain-simulator/transactions.cs-e2e.ts

```diff
@@ -0,0 +1,291 @@
+import axios from "axios";
+import { ChainSimulatorUtils } from "./utils/test.utils";
+import { config } from "./config/env.config";
+
+describe('Transactions e2e tests with chain simulator', () => {
+  beforeAll(async () => {
+    await ChainSimulatorUtils.waitForEpoch(2);
+  });
+
+  describe('GET /transactions', () => {
+    it('should return status code 200 and a list of transactions', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions`);
+      expect(response.status).toBe(200);
+    });
+
+    it('should support pagination', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?from=0&size=1`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+    });
+
+    it('should return transactions with sender filter applied', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?sender=${config.aliceAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThan(1);
```

Comment:
check that all the transactions contain the provided sender

Reply:
done